packages/app/src/layouts/default.vue

```diff
@@ -48,6 +49,13 @@
           <component :is="Component" />
         </transition>
       </router-view>
+      <CloudConnectModals
+        v-if="showConnectDialog && query.data.value"
+        :show="showConnectDialog"
+        :gql="query.data.value"
+        @cancel="showConnectDialog = false"
+        @success="showConnectDialog = false"
+      />
```

Comment:
Ah, looked at this a bit deeper and it wasn't quite doing everything I thought it was - we've still got multiple instances of `CloudConnectModals`. Just thinking out loud here, no action expected -I wonder whether it would be viable to consolidate to the single instance and control via new global state as part of the upcoming work we're doing where we'll be touching these modals again, or whether that would make things a bit too tightly coupled thru state 🤔